.shellcheckrc

```diff
@@ -0,0 +1 @@
+disable=SC1090,SC1090 # Ignore warnings about being unable to follow sourced files
```

Comment:
```suggestion
disable=SC1090 # Ignore warnings about being unable to follow sourced files
```

Is there a reason to put it in twice?

Reply:
Ah, no - good catch. Actually the second instance should be `SC1091`

Reply:
FP'd to fix